src/dashboard/components/Deal.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * WordPress dependencies.
+ */
+import { __ } from '@wordpress/i18n';
+
+const Deal = ( props ) => {
+	return (
+		<div className="otter-deal">
+			<a href={props.link}>
```

Comment:
All looks good, we can just make sure the banner opens in a new tab. @Soare-Robert-Daniel You can make the small change and put it for QA & ping Harshit to take a look.